src/Client/app/(root)/my-exams/[id]/page.tsx

```
@@ -1,351 +1,415 @@
+            <Logo />
+          </div>
+          <div className="flex gap-4 rounded-full border-small border-default-200/20 bg-background/60 px-4 py-2 shadow-medium backdrop-blur-md backdrop-saturate-150 dark:bg-default-100/50">
+            <div>
+              Questions Left: {questionsLeft}/{codingQues + regularQues}
+            </div>
+            <div className="flex items-center gap-1 before:content-[''] before:w-2 before:h-2 before:bg-red-500 before:rounded-full">
+              <p>Time Left : </p>
+              <p
+                className={`font-mono ml-1 ${timeLeft !== undefined &&  timeLeft< 300 ? "text-danger" : "text-success"}`}
+              >
+                {timeLeft !== undefined ? FormatTimeHourMinutesSeconds({ seconds: timeLeft }) + ' s' : 'Loading...'}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)